src/ngraph/runtime/gpu/gpu_cuda_kernel_builder.cpp

```diff
@@ -326,9 +326,9 @@ void runtime::gpu::CudaKernelBuilder::get_softmax_op(codegen::CodeWriter& writer
             // loop through all reduction axis
```

Comment:
nit: refactor `:: get_softmax_op` to use the same lambda callbacks as `:: get_softmax_block_reduce_op`